tests/e2e/tools/lib/rest.js

```diff
@@ -0,0 +1,40 @@
+import { nonce } from './nonce';
+
+async function rest(page, context, options) {
```

Comment:
Looks like we are passing the page param to this function but not using it, is there a reason?